.gitignore

```diff
@@ -23,7 +23,7 @@ out/
 build/
 
 # Local configuration file (sdk path, etc)
-local.properties
+#local.properties
```

Comment:
@gianghv-3456 
? sao cmt dòng này lại vậy em

Reply:
Sau khi em add GEO API KEY vào dù giấu key trong localproperties nhưng ktlint check báo "not found file local.properties" nên em đẩy local.properties không có key lên để pass ktlint